### DISCUSSION TITLE
Small responsive image preview is in blur mode

### DISCUSSION THREAD

**Discussion by ayman-elmalah** [spatie/laravel-medialibrary#2436] [Q&A]
I implemented responsive images using this  package,

When small image in preview it look like a blur image

can i know how to solve it, i'm just using the $model->img() to preview image

![image](https://user-images.githubusercontent.com/22693197/117544908-d3e0c880-b023-11eb-878d-8d278d1bf88c.png)

**Comment by obrunsmann** [comment]
Is there any solution out there? Having the same issue... it works randomly some times after clearing the cache, but then stops again few days or hours later. Some users report that it never works.
